modules/lib/lib-portal/src/main/java/com/enonic/xp/lib/portal/current/GetCurrentSiteConfigHandler.java

```diff
@@ -41,7 +40,7 @@ public PropertyTreeMapper execute()
             }
             else
             {
-                final Project project = this.projectService.get().get( ProjectName.from( ContextAccessor.current().getRepositoryId() ) );
+                final Project project = this.projectService.get().get( ProjectName.from( this.request.getRepositoryId() ) );
```

Comment:
I would not do it in this PR. It is a tricky topic - where to get project from.
This change introduces ambiguity - project from request, but branch from context